src/Consolonia.Core/Controls/FileOpenPicker.axaml.cs

```diff
@@ -1,15 +1,19 @@
 using System;
+using System.Collections.Specialized;
 using Avalonia;
 using Avalonia.Controls;
 using Avalonia.Input;
 using Avalonia.Interactivity;
 using Avalonia.Platform.Storage;
+using Avalonia.Threading;
 using Iciclecreek.Avalonia.WindowManager;
 
 namespace Consolonia.Core.Controls
 {
-    internal partial class FileOpenPicker : ManagedWindow
+    internal partial class FileOpenPicker : ManagedWindow, IDisposable
```

Comment:
But Dispose is never called for this dialogs https://github.com/jinek/Consolonia/blob/be82d91a71ed8968bd23e2200eaea827ea4a55ed/src/Consolonia.Core/Infrastructure/ConsoloniaStorageProvider.cs#L27